### DISCUSSION TITLE
Star a new instance with shuffle disabled?

### DISCUSSION THREAD

**Discussion by 097115** [Beinsezii/ompl#4] [Q&A] [answered]
Let's say I start `ompl` this way:

    ompl main ~/Music -f artist album -s "<track|<track> - ><title>"

Now, can I somehow add `shuffle false` in the mix? Or disable it in any other way from the very start?

Sure, pressing `r` isn't really hard. But one needs to remember it :)

**Comment by Beinsezii** [comment, marked as answer]
Yea I forgot to add that. Fixed in release 0.7.0

**Comment by 097115** [comment]
@Beinsezii 

Thanks a lot!

And sorry for some sort of belated comment but would it be possible to add something like `--noloop`, too?

**Comment by Beinsezii** [comment]
Currently there's no engine support to disable looping.

Right now I'm rewriting the whole backend which might affect that, so not sure of an ETA but I'll write it down as a TODO.

**Comment by 097115** [comment]
Thanks again!

**Comment by Beinsezii** [comment]
Implemented `--norepeat` + more in fb2228026fba00d0af4d4edec7317e4cfce4b1a6

Feel free to try out the latest master. It builds with the new backend by default, which I'm also daily-driving.

Also repeat is now bound to `r` instead of shuffle, which is `e` now.

**Comment by 097115** [comment]
You rock! It works, thanks :)
